django/contrib/admin/static/admin/css/forms.css

```diff
@@ -303,8 +303,10 @@ body.popup .submit-row {
     padding: 10px 15px;
     height: 15px;
     line-height: 15px;
-    margin: 0 0 0 5px;
+    margin: 0 5px 5px 5px;
     color: var(--button-fg);
+    vertical-align: top;
```

Comment:
Do we need this? Looks unnecessary.

Reply:
Not really. It's needed if the input element is near only. But there are no such cases for now. I'll just remove it.